config/routes/api/v2.rb

```diff
@@ -318,7 +318,11 @@ class ActionDispatch::Routing::Mapper
             end
           end
 
-          api_resources :upstream_subscriptions, only: :index
+          api_resources :upstream_subscriptions, only: [:index] do
+            collection do
+              delete :destroy
```

Comment:
why not use the shorthand? DELETE is the default method for a destroy action.

Reply:
I'm not destroying a single resource - but a collection of them - so there won't be an "upstream subscription ID" in the URL. I think this is the correct way...

Reply:
Ah yeah, right. :)